src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -440,3 +440,38 @@
         visibility: hidden;
     }
 }
+
+/***** Error Toolbar *****/
+.sfErrorToolbar {
```

Comment:
can't you apply the `sf-toolbarreset` class on your error toolbar too ? This would avoid duplicating many things (your error toolbar is missing the resetting of box-sizing for instance, and a few other things).

This would also handle the positioning automatically.